### ISSUE TITLE
[EPIC] OpenPulse Documentation

### ISSUE THREAD

**Issue by lcapelluto** [Qiskit/qiskit-terra#3676]
Each item in the checklist needs its own issue, and that issue needs to be linked here before starting.

### Terra

 - [x] getting started
 - [ ] creating instructions - IP
    - [ ] What are channels?
    - [ ] building pulses
        - [ ] using the `pulse_lib` or using `numpy`
        - [ ] using parametric pulses (this method should be encouraged)
    - [ ] What are frames? What is a frame change? How to add one? Why might you add a FC on derived channels? This ties into #3467 also
        - [ ] How to update the modulation frequency of a channel (or SetFrequency, later)
 - [x] how to build a schedule (explain the syntax, explain multiple ways to build the schedule, explain schedule operations and composition)
 - [ ] how to add measurements, including explaining `meas_map` with a link to page about `meas_level`s (using a custom discriminator?) - IP
 - [ ] how to use execute
    - [ ] special args: frequencies currently passed that way, until #3467 is done
    - [ ] how to use the pulse simulator
 - [ ] more advanced schedule building
    - [ ] what are control channels? how to get data about them and how to use them?
 	- [ ] how to build and understand the system hamiltonian (hopefully, can mostly link to other resources)
 - [ ] getting data about the system
    - [ ] configuration
    - [ ] defaults and InstructionScheduleMap
    - [ ] properties
 - [ ] Using the scheduler (default scheduling method, demo how to use it, explain key features such as acquisition alignment)
    - [ ] different methods
    - [ ] custom scheduling with InstMap
    - [ ] extending the scheduler
 - [ ] special results types
    - [ ] different meas levels
    - [ ] extracting data from the results objects for each meas level
 - [ ] how to make a new ParametricPulse
 - [ ] help with typical use cases
    - [ ] how to vary parameters
 - [ ] physics (this section could potentially just be a list of references, it's important that the references exist and are easy to find, but explaining physics is not the aim of software documentation)
    - [ ] how does the system evolve under a pulse
 - [ ] troubleshooting: this is stuff that people ask about over time. Hopefully, everything will have been covered in the above, but there should still be a landing spot for this

### Aer
 - [ ] Using the pulse simulator - IP

## Tutorials for OpenPulse should demonstrate:
 - frequency sweep
 - basic implementation of each Pauli gate using pulse
 - basic CX bringup tuning CR pulses
 - measuring t1
 - measuring t2
 - dynamical decoupling
 - qubit spec (chevron)
 - creating a custom discriminator

Most of these are done in various places, and may need cleaning up, updating to use the latest code, or more thorough explanations.

**Comment by lcapelluto** [comment]
user case misconception: framechange can be used to rotate around y axis

**Comment by lcapelluto** [comment]
@DanPuzzuoli tracking your tutorial contributions here :D

**Comment by lcapelluto** [comment]
Moved here: https://github.com/Qiskit/qiskit-iqx-tutorials/issues/869
